packages/reports/src/sass/app/modules/smart-chart.scss

```diff
@@ -316,6 +316,10 @@
     }
 }
 
+.smartcharts .barrier-area {
+    z-index: 0 !important;
```

Comment:
For all barriers with shadow type not equal to NONE_DOUBLE, z-index is 0. But for mentioned type, z-index is 101 and written inline in SmartCharts. NONE_DOUBLE is applied only for Lookbacks High-Low (packages/shared/src/utils/constants/barriers.ts) and for contract with barrier_cont === 2. At the moment it's also Lookbacks High-Low .
As it's a temporary design solution and in future we will show not barriers, but spot markers, I decided to add small css fix in this rep, not in SmartCharts.

Reply:
@kate-deriv what issue is it fixing? is it the issue with the barrier being on top of contract markers? if so, it's better to not fix it in this PR because @henry-deriv will work on it separately🙏🏻

Reply:
please dont make changes to smartchart styles from deriv-app if possible. There are cases where styles don't get applied.

Reply:
Styles above was for fixing another problem, different from what Henry did. 
But in order not to rewrite SmartChart styles like this, I pass new z-index via config.

Reply:
The problem was, that upper barrier (if it has `NONE_DOUBLE` type in config) has inline style (`z-index: 101`), which was written in SmartChart. Because of such high z-index value, upper barrier was overlapping markers (for entry and exit spot).